src/ng/http.js

```diff
@@ -426,7 +426,7 @@ function $HttpProvider() {
    *   }]);
    * ```
    */
-  var xsrfWhitelistedOrigins = this.xsrfWhitelistedOrigins = [];
+  var xsrfTrustedOrigins = this.xsrfWhitelistedOrigins = this.xsrfTrustedOrigins = [];
```

Comment:
How come this didn't trigger a solidarity warning?

Reply:
Ii looks like it did? https://github.com/angular/angular.js/pull/17074/checks?check_run_id=1156542750

I believe that if you make a change to a line that uses any of the terms, but don't remove the usage of the term, it warns, but does not error.